### ISSUE TITLE
More control over importing behavior

### ISSUE THREAD

**Issue by kbrandwijk** [ardatan/graphql-import#16]
Proposal (based on #13, #14 and #15):

Add optional parameter to `importSchema`:
```typescript
rules? : {
   removeUnusedTypes?: boolean = true,
   importAllInterfaceImplementations?: boolean = false,
   importMissingTypes?: boolean = true,
   orderTypes?: boolean = false,
   strict?: boolean = true,
   validate?: boolean = false
}
```
Most are self-explanatory, `strict: true` would fail on missing schema files or missing imported types, otherwise silently ignored. `validate: true` would run the linter.

**Comment by schickling** [comment]
That looks good. Would be great if you could update your proposal to include default values.

**Comment by kbrandwijk** [comment]
@schickling Done.

**Comment by ccnixon** [comment]
Hey guys - any plans on taking a look at this in the near term? My use case:

I'm hoping to use [graphql-s2s type inheritance](https://github.com/nicolasdao/graphql-s2s#type-inheritance) in tandam with import statements but the validation step is failing. It would be great to be able to set a flag to bypass the schema validation step and and just compile the schema. Thanks!

**Comment by raeesaa** [comment]
I am having similar use case as @ccnixon. It would be great to have an option to bypass schema validation.

**Comment by raeesaa** [comment]
@kbrandwijk Any updates on this?

**Comment by DavidStummer** [comment]
@kbrandwijk we would love this too

**Comment by ardatan** [comment]
Available in 1.0.0!
